_posts/2018-05-21-figure-4-0-0.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: Release of OMERO.figure 4.0.0
+intro-blurb: The OME team is pleased to announce the release of OMERO.figure 4.0.0
+---
+We are pleased to announce the release of OMERO.figure 4.0.0.
+
+This is a major release that adds support for "big" tiled images.
+It includes changes to the way ROI stroke-widths are displayed.
+Previously, ROI stroke-widths were measured in image pixels so that
+stroke-widths decreased as an image panel was zoomed out.
+However, this meant that lines of similar width can appear at different
+thicknesses on different images. It also meant that on "big" images that
+were zoomed out a long way, lines would disappear completely.
+
+Now, stroke-widths are measured by page points so all lines of a
+similar stroke-width will appear the same thickness on the page,
+independent on the size or zoom of the panel they are on.
+
+When older figures are opened with OMERO.figure 4.0.0, stroke-widths
+will be updated so that the appearance of ROI lines remains the same.
+
+Other features:
+
+* Allow loading Polygons from OMERO to add to images
+* Open-with Figure and adding Images now preserves the order of images
```

Comment:
There is an initial capital for one "Images" but not for any of the other images in the list: are those special?